.github/workflows/ci-arm64.yml

```
@@ -26,6 +31,7 @@ jobs:
 
   rust-lint:
     name: Rust Lint
+    if: false  # ARM64 pipeline disabled
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)